xclim/indices/_synoptic.py

```diff
@@ -42,31 +43,38 @@ def jetstream_metric_woolings(
     """
     lon_min = -60
     lon_max = 0
-    lons_within_range = any((ua["lon"] >= lon_min) & (ua["lon"] <= lon_max))
+    lons_within_range = any(
+        (ua.cf["longitude"] >= lon_min) & (ua.cf["longitude"] <= lon_max)
+    )
     if not lons_within_range:
         raise ValueError(
             "Longitude values need to be in a range between %s-%s. Consider changing the longitude coordinates to between -180.E–180.W"
             % (lon_min, lon_max)
         )
```

Comment:
Should lon_min, lon_max be a function argument? And, should this check be put into another function (i.e. following SRP)?

Reply:
Having those as arguments seems logical to me, unless other values do not make sense for the algorithm?
A thing that can be done to ensure a meaningful indicator is too have them are arguments on the_indice_ but inject the normal values in the _indicator_. This way, the users is able to control it, but only when using the less-rigid version.

Reply:
I am unsure how the *indicator* part of the module exactly works, so I am not sure how to implement this. I believe this is related to why f5b7fbbe66ff6d64c7b9e493698fb5d27cb9c07d is failing. Happy to learn, so I can be more help, but alternatively, happy for anyone to take over and fix!